tensorflow/contrib/sparsemax/python/ops/sparsemax_loss.py

```diff
@@ -57,7 +57,7 @@ def sparsemax_loss(logits, sparsemax, labels, name=None):
     # Use a conditional where instead of a multiplication to support z = -inf.
     # If z = -inf, and there is no support (sparsemax = 0), a multiplication
     # would cause 0 * -inf = nan, which is not correct in this case.
-    sum_s = array_ops.where(
+    sum_s = array_ops.where_v2(
```

Comment:
The tests for this are failing because they rely on the old broadcasting behavior of tf.where. Please fix.